github/add_security_md_to_json_list.py

```diff
@@ -157,7 +157,7 @@ def add_commit_push_security_md(repo_path, branch_name):
 
 # Create a pull request
 def create_pull_request(repo_path, branch_name, default_branch):
-    """Create a pull request for the branch, attempt to add reviewers, and assign 'wz-gsa'."""
+    ""f"Create a pull request for the branch, attempt to add reviewers, and assign '{ASSIGNEE}'."""
```

Comment:
With the `f` in the middle of the quotes, you no longer have a triple-quoted string, you have three strings being silently concatenated.

This should probably be either:
`f"""Create a pull request for the branch, attempt to add reviewers, and assign '{ASSIGNEE}'."""`

or:
`f"Create a pull request for the branch, attempt to add reviewers, and assign '{ASSIGNEE}'."` (since the triple-quotes aren't actually necessary here)